alerts_backend/python/app.py

```diff
@@ -1,15 +1,30 @@
 """Query alert information from AeroAPI and present it to a frontend service"""
 import os
 from datetime import datetime
-from typing import Dict, Any, Union
+from typing import Dict, Any, Tuple
 
 import json
 import requests
 from flask import Flask, jsonify, Response, request
+from flask.logging import create_logger
 from flask_cors import CORS
 
 from sqlalchemy import (exc, create_engine, MetaData, Table,
-                        Column, Integer, Boolean, Text, insert, Date)
+                        Column, Integer, Boolean, Text, insert, Date, DateTime)
+from sqlalchemy.sql import func
+from sqlalchemy import (
+    exc,
+    create_engine,
+    MetaData,
+    Table,
+    Column,
+    Integer,
+    Boolean,
+    Text,
+    insert,
+    Date,
+    select,
+)
```

Comment:
It looks like this new import statement is partially redundant with what's above starting at line 12. Can you please combine the two into a single import statement?